release-process.rst

```diff
@@ -225,6 +225,14 @@ Next, build the 32- and 64-bit Windows binaries:
    $ ninja
    $ cpack
 
+Create a tarball:
+
+.. code::
+
+  $ git archive -o uncrustify-0.1.2.tar.gz `git rev-parse HEAD`
```

Comment:
Will discuss this more over private mail...

At any rate, I would lose the "test it somewhere" and instead replace the `wget` with something like "Using the tarball you created earlier:". Also, I wonder if `git rev-parse HEAD` should instead be `uncrustify-0.1.2`, i.e. use the tag explicitly? (Hopefully they're the same SHA! ...but...)